src/main/scala/li/cil/oc/integration/igwmod/ModIGWMod.scala

```diff
@@ -0,0 +1,138 @@
+package li.cil.oc.integration.igwmod
+
+import cpw.mods.fml.common.eventhandler.SubscribeEvent
+import igwmod.api.{ItemWikiEvent, BlockWikiEvent}
+import li.cil.oc.OpenComputers.log
+import li.cil.oc.common.item._
+import li.cil.oc.common.block._
+import net.minecraftforge.common.MinecraftForge
+
+object ModIGWMod {
+  def init() {
+    log.info(s"IGW-Mod integration loading")
+    MinecraftForge.EVENT_BUS.register(this)
+  }
+
+  @SubscribeEvent
+  def onPageRequest(event: BlockWikiEvent) {
```

Comment:
Create a new class (ideally a singleton, i.e. object) for these, put them in there. The way it is now, the class loader will error if IGW isn't present.